app/Models/Version.php

```diff
@@ -142,4 +147,11 @@ public function imageDerivativeDirectoryPath(): string
     {
         return sprintf('%s/%s', $this->Media->baseDirectory(), $this->getKey());
     }
+
+    public function hash(): Attribute
+    {
+        return Attribute::make(
+            get: fn(): string => md5(sprintf('%s-%s', $this->number, $this->created_at))
```

Comment:
isnt md5 super dead?

Reply:
it's faster than sha256 and security is not a problem here, is it?